doc/api.raml

```diff
@@ -2833,6 +2887,110 @@ types:
                       }
                     }
                   }
+        404:
+          body:
+            application/vnd.api+json:
+              type: error_response
+              example: |
+                {
+                  "message": "Record Not Found",
+                  "errors": [
+                    {
+                      "id": "ActiveRecord::RecordNotFound",
+                      "code": "1141",
+                      "status": "404",
+                      "title": "Record not fund",
+                      "detail": "No matching address record were found",
+                      "href": "http://shaojunda.coding.me/doc/api_doc.html#statistics__info_name__get"
```

Comment:
Can this be replaced by official link ?

Reply:
thanks, updated: https://github.com/nervosnetwork/ckb-explorer/pull/491/commits/5cac424251eea2c7b831fe0db28880a651593da0